cmd/gaia/app/genesis_port.go

```diff
@@ -0,0 +1,85 @@
+package app
```

Comment:
IMHO this script should really live under `scripts/`. In either case, it definitely should not live here.

Reply:
Agree with @alexanderbez that it should be a separate command.

Reply:
updated